politeiad/backendv2/tstorebe/plugins/dcrdata/dcrdata.go

```diff
@@ -255,6 +255,9 @@ func New(settings []backend.PluginSetting, activeNetParams *chaincfg.Params) (*d
 	case chaincfg.TestNet3Params().Name:
 		hostHTTP = dcrdata.SettingHostHTTPTestNet
 		hostWS = dcrdata.SettingHostWSTestNet
+	case chaincfg.SimNetParams().Name:
+		hostHTTP = dcrdata.SettingHostHTTPSimNet
+		hostWS = dcrdata.SettingHostWSSimNet
```

Comment:
With simnet, you're gonna have self-signed certificates running dcrdata's web server, so either skip verification or introduce a dcrdatacert option to append to the client's root certificates.  To skip verification, it looks like about 25 lines down, you could pass `true` to `util.NewHTTPClient` when on simnet.